gfx/MShockXotto+/pngs_large_64x80/mods/DinoMod/monster/mon_pachyrhinosaurus/mon_pachyrhinosaurus.json

```diff
@@ -0,0 +1,8 @@
+{
+  "id": "mon_pachyrhinosaurus",
```

Comment:
You don't need the json if the id and the name of the sprite are the same

Reply:
Thanks, I started without and then added to troubleshoot an unrelated issue. Good instinct, wrong direction